Assets/Scripts/Gameplay/AI/StateMachine/States/AttackingAIState.cs

```diff
@@ -31,6 +31,7 @@ public async Task<StateResult> Launch()
             {
                 if (!_observeArea.HasTarget)
                 {
+                    _observeArea.DeactivateAttackCollider();
```

Comment:
Move this line to and of method before return

Reply:
Done